raft/node.go

```diff
@@ -562,7 +575,7 @@ func (n *node) ReadIndex(ctx context.Context, rctx []byte) error {
 func newReady(r *raft, prevSoftSt *SoftState, prevHardSt pb.HardState) Ready {
 	rd := Ready{
 		Entries:          r.raftLog.nextUnstableEnts(),
-		CommittedEntries: r.raftLog.nextCommittedEnts(true /* allowUnstable */),
```

Comment:
Ok, disregard my earlier comment on this then, I understand now that `true` was the right setting for the status quo (no async writes). I didn't realize (but now that I actually think about it remember) that we do emit unstable entries for application. I just spent time on that in the context of single-voter nodes.